src/sidebar/components/menu.js

```diff
@@ -21,6 +32,80 @@ const menuArrow = className => (
  */
 let ignoreNextClick = false;
 
+function isElementVisible(element) {
+  return element.offsetParent !== null;
+}
+
+function MenuList({ align, menuButtonId, contentClass, closeMenu, children }) {
+  const menuRef = useRef(null);
+
+  const handleMenuKeyDown = event => {
+    const menuItems = Array.from(
+      menuRef.current.querySelectorAll('[role^="menuitem"]')
```

Comment:
The `[role^=menuitem]` selector here matches child roles of `menuitem` such as `menuitemradio`.